admin/my-yoast-proxy.php

```diff
@@ -7,20 +7,44 @@
  * @package WPSEO\Admin
  */
 
+$plugin_version = filter_input( INPUT_GET, 'plugin_version', FILTER_SANITIZE_STRING );
```

Comment:
It looks like this variable is only used in the case statement (line 20 - 25 ). I would suggest to move it there.

Reply:
When the `switch` statement gets expanded it will probably also have a fallback that should use the `plugin_version` and `suffix`.

Reply:
Ah okay 👍